src/components/NavBar/NavBar.js

```diff
@@ -16,8 +19,27 @@ class NavBar extends Component {
     </Tooltip>
   );
 
+  showCurrentPursuance = (pursuances) => {
+    let id = parseInt(window.location.pathname.slice(-1), 10);
```

Comment:
@thevaleriemack You can get the pursuance ID from `this.props.match.params.pursuanceId`, like this: https://github.com/PursuanceProject/pursuance/blob/6a8af04a6b85e26f3af7de8a188ac8e822d74daf/src/components/Content/Pursuance/views/DiscussView.js#L16-L24

Reply:
Not sure how to do this. "match" is coming up undefined I think because Navbar isn't being rendered using Route

<img width="563" alt="screen shot 2018-03-14 at 8 48 12 am" src="https://user-images.githubusercontent.com/8445610/37413729-de9d5aea-2764-11e8-9721-5f06004dc967.png">

Reply:
I can try getting PursuancePage to push its state up to Navbar? Not sure how that works with redux but I'll give it a shot.

Reply:
I see. Rendering `NavBar` using Route is probably the way to go, it seems to me. Luckily everything being passed to it manually is in Redux anyway, so how about reduxifying NavBar then having Route render it on every page?

Reply:
Finally got this to work!

match is populated but the match is not exact because NavBar path is always root "/" so the params end up empty even when on PursuancePage.

Now I'm using location passed by react router but I still have to slice the pathname to get the id.

currentPursuanceId won't work either because it doesn't change when moving from a pursuance page to a non pursuance page like Dashboard.

<img width="558" alt="screen shot 2018-03-25 at 2 26 30 pm" src="https://user-images.githubusercontent.com/8445610/37880127-8f9f41ec-3038-11e8-889a-1dc1f50f7982.png">